src/__tests__/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Value, withValue } from '..';
+
+function render(spy) {
+  return function(value, onChange) {
```

Comment:
If you recall how I had designed the API before, all I needed to do to make this test work was bring it inline with this (I'm assuming you still want to keep it this way).